src/app/core/learning-object-module/bundling/bundling.service.ts

```diff
@@ -39,4 +48,13 @@ export class BundlingService {
       return throwError(error);
     }
   }
+
+  downloadObject(event: MouseEvent, object: LearningObject, index: number) {
```

Comment:
i don't know if this should belong here, this logic was meant for `library.component` to be used in junction with `downloadBundle`. if we're gonna move this over it should be `libraryService.downloadBundle` that should be moved here and refactored so it can be used across components.